inventory/sample/hosts.yml

```diff
@@ -0,0 +1,19 @@
+vm:
+  hosts:
+    os-builder:
+      ansible_port: 22
+      ansible_host: 10.182.31.179
+      ansible_user: ubuntu
+master:
+  hosts:
+    ubuntu:
+      ansible_port: 5555
+      ansible_host: 10.182.31.179
+      ansible_user: rpi_admin
+      rpi_static_ip: 10.0.0.10
+node:
+
```

Comment:
could you provide 1 node as well?

Reply:
For the mean time the playbook can generate only one image at a time.

Reply:
As per our discussion, let's try to use a combination of `serial: 1`and `delegate_to: vm` to allow processing as many hosts as needed.
Ideally we would like to re-use the Qemu machine between hosts for performance optimization.

Reply:
Let this be part of another PR.